books/free-programming-books-bn.md

```diff
@@ -72,3 +73,7 @@
 
 * [এসকিউএল পরিচিতি(SQL Introduction in Bangla)](https://www.sattacademy.com/sql/sql-intro.php) - Satt Academy
 
+### Reactjs
+
+* [reactjs ভূমিকা] (https://youtu.be/CiOivSO25JY)
```

Comment:
No shortened links please. Also please read docs/CONTRIBUTING.md to check all the spacing info and author attribution

Reply:
Also this is a course please move it to a course file